routes/v1.js

```diff
@@ -284,34 +241,29 @@ api.declare({
     'Delete a WorkerType definition, submits requests to kill all ',
     'instances and delete the KeyPair from all configured EC2 regions',
   ].join('\n'),
-}, function(req, res) {
+}, async function(req, res) {
   var workerType = req.params.workerType;
 
   if(!req.satisfies({workerType: workerType})) { return undefined; }
 
-  var p = this.WorkerType.load({workerType: workerType});
+  try {
+    var wType = await this.WorkerType.load({workerType: workerType});
+    await wType.remove();
```

Comment:
This is fine... Performance is hardly important. But it would be faster to use:
https://github.com/taskcluster/taskcluster-base/blob/cd8b589137be779d2c039ede767f9f4a24ef1863/lib/entity.js#L842-L850